apps/integrator-api/src/routes/executor/index.ts

```
@@ -225,6 +228,12 @@ export default async function executor(app: FastifyInstance) {
         throw app.httpErrors.unauthorized("Unauthorized!");
       }
 
+      if (!process.env.INTEGRATOR_URL) {
```

Instead of checking here, shall we check for the `INTEGRATOR_URL` variable in the fastify-env schema? I'm not a big fan of checking env vars at runtime, it can become really messy. It would be better if we just let the app crash if we're missing the env var

---

Done!